Mathlib/MeasureTheory/Integral/IntegrableOn.lean

```diff
@@ -78,6 +78,7 @@ theorem hasFiniteIntegral_restrict_of_bounded [NormedAddCommGroup E] {f : α →
   hasFiniteIntegral_of_bounded hf
 
 variable [NormedAddCommGroup E] {f g : α → E} {s t : Set α} {μ ν : Measure α}
+  [TopologicalSpace ε] [ContinuousENorm ε]
```

Comment:
These variables are never actually included yet, right?
Isn't the variable `[TopologicalSpace ε]` declared earlier still in scope?
I think this will give linter errors as soon as any theorem actually uses `ε`.
I didn't actually realize that definitions still only included the variables that they actually used, not all variables.

Reply:
Good point about the shadowing: I moved the earlier hypotheses into a separate section.

The definition IntegrableOn includes the ContinuousENorm hypothesis.